commit.go

```diff
@@ -321,7 +322,8 @@ func Push(image string, dest types.ImageReference, options PushOptions) error {
 		return errors.Wrapf(err, "error recomputing layer digests and building metadata")
 	}
 	// Copy everything.
-	err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter))
+	//TOM: todo add credsContext
```

Comment:
Removed.  Not needed before merging.  This will be turned on when the cred params are put into the 'buildah commit' command in the next PR for buildah auth.